sormas-api/src/main/resources/strings.properties

```diff
@@ -1151,4 +1152,4 @@ DefaultPassword.ownUserNewPasswordSetHints=Your account has been successfully up
 DefaultPassword.otherUsersNewPasswordSetHints=Your accounts have been successfully updated. Make sure to copy the new passwords and to notify the affected users about the password change. The new passwords are displayed only once.
 DefaultPassword.newPassword=New Password
 DefaultPassword.newPasswordPlaceholder=New Password: %s
-DefaultPassword.unchanged=- unchanged -
\ No newline at end of file
+DefaultPassword.unchanged=- unchanged -
```

Comment:
There is an unnecessary newline added

Reply:
It is done by my autoformatter, so I assume that is desired. Am I wrong here?

Reply:
Assuming you are using IntelliJ, the newline option is actually independent from the code style settings. It can be found under File->Preferences->Editor->General->Ensure every saved file ends with a line break.

Reply:
Ah okay, didn't know that. Thank you.